test/e2e/aks_tags.go

```diff
@@ -98,16 +98,11 @@ func AKSAdditionalTagsSpec(ctx context.Context, inputGetter func() AKSAdditional
 			}
 		}
 
-		By("Deleting all tags for control plane")
```

Comment:
@nojnhuh does this need to be re-added when #4248 merges?

Reply:
It doesn't necessarily need to be, but it could. This should also be added to the node labels and node taints tests, so I was planning on doing that all in one follow-up PR (next week) unless we want that in sooner.

Reply:
Do we have a follow-up issue for labels and taints? It wouldn't hurt to have that test signal when we merge this one IMO, if we're blocking it on v2.4 anyways but works either way.

Reply:
We do now: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/4266.